File: TransCoder_tokenized_test_set_functions/cpp/FIND_THE_LARGEST_THREE_ELEMENTS_IN_AN_ARRAY.cpp
void find3largest ( int arr [ ], int n ) {
  sort ( arr, arr + n );
  int check = 0, count = 1;
  for ( int i = 1;
  i <= n;
  i ++ ) {
    if ( count < 4 ) {
      if ( check != arr [ n - i ] ) {
        cout << arr [ n - i ] << " ";
        check = arr [ n - i ];
        count ++;
      }
    }
    else break;
  }
}

File: TransCoder_tokenized_test_set_functions/python/FIND_THE_LARGEST_THREE_ELEMENTS_IN_AN_ARRAY.py
def find3largest ( arr, n ) :
    arr = sorted ( arr )
    check = 0
    count = 1
    for i in range ( 1, n + 1 ) :
        if ( count < 4 ) :
            if ( check != arr [ n - i ] ) :
                print ( arr [ n - i ], end = " " )
                check = arr [ n - i ]
                count += 1
        else :
            break
    
